src/components/cylc/workflow/Toolbar.vue

```diff
@@ -285,6 +372,16 @@ export default {
     },
     isStopped () {
       this.expecting.stop = null
+    },
+    nWindow (newVal) {
+      // the user has requested to change the window size
+      this.setGraphWindow(newVal)
+    },
+    nEdgeDistance (newVal) {
+      // the scheduler has reported that the window size has changed
+      if (newVal != undefined) {
```

Comment:
Oh, for some reason ESLint only allows non-strict equality comparison with `null` not `undefined` (https://github.com/eslint/eslint/issues/13537)
```suggestion
      if (newVal != null) {
```

Reply:
(If using `!= null` you don't have to worry about whether it's `null` or `undefined`)